infra/kubernetes/main.tf

```diff
@@ -0,0 +1,88 @@
+// terraform config //
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+  # backend "s3" {
+  #   bucket  = "cncf-credits-infra-kubernetes-tfstate"
+  #   key     = "terraform.tfstate"
+  #   region  = "ap-southeast-2"
+  #   encrypt = true
+  # }
+}
+
+provider "aws" {
+  profile = "default"
+  region  = "ap-southeast-2"
```

Comment:
why this region ? possible to use `us-east-1` ?

Reply:
@sftim some services we want to use are only available in `us-east-1`. As a daily AWS user, I know the pain. :-)

Reply:
I'm sure it'd be fine to switch the region